lib/backend/backend.go

```diff
@@ -433,6 +433,12 @@ func CreateRevision() string {
 	return uuid.NewString()
 }
 
+// BlankRevision is a placeholder revision to be used by backends when
+// the revision of the item in the backend is empty. This can happen
+// to any existing resources that were last written before support for
+// revisions was added. The value is the base32 encoding of deadbeef.
+const BlankRevision = "GB4GIZLBMRRGKZLG"
```

Comment:
If there is a bug, it will be non-obvious that this is a sentinel value.  I'd suggest going with something that is more obviously not a real revision.  Ideally something that contains at least one character that ins't allowable in any real revision encoding (e.g. `@empty`).

Reply:
The only backends where this is relevant also use UUIDs for revisions (i.e. they use `backend.CreateRevision`). Maybe we could use `00000000-0000-0000-0000-000000000000`? It's never generated as a uuid v4, so it's actually impossible (as opposed to practically impossible) for it to be a generated revision of an item after the introduction of revisions.